lib/middleware/request_tracker.rb

```diff
@@ -8,6 +8,16 @@ class Middleware::RequestTracker
   @@detailed_request_loggers = nil
   @@ip_skipper = nil
 
+  # You can add exceptions to our app rate limiter in the app.yml ENV section.
+  # example:
+  #
+  # env:
+  #   DISCOURSE_MAX_REQS_PER_IP_EXCEPTIONS: >-
+  #     14.15.16.32/27
+  #     216.148.1.2
+  #
+  @@static_ip_skipper = ENV['DISCOURSE_MAX_REQS_PER_IP_EXCEPTIONS']&.split&.map { |ip| IPAddr.new(ip) }
```

Comment:
I think this might be cleaner as a GlobalSetting, rather than directly accessing `ENV`? IIRC that's how we do things like this elsewhere.

Reply:
I tried that at first, but GlobalSetting isn't available at the time this is set :(. Open to ideas on how to lazily set that in more ergonomic way.

Reply:
> GlobalSetting isn't available at the time this is set

Ahh I see, makes sense :+1:

Reply:
> Open to ideas on how to lazily set that in more ergonomic way.

Perhaps we can move it into a method and memoize the value using a class variable.

Reply:
Also, I'm wondering why we need to use a class variable here when a constant would work. That way, we can easily stub out the constant in our tests and do no need the two methods declared below for testing purposes.